lark/lark.py

```diff
@@ -115,6 +119,8 @@ class LarkOptions(Serialize):
         'edit_terminals': None,
         'g_regex_flags': 0,
         'use_bytes': False,
+        'import_sources': [],
```

Comment:
The common name for this is `path`, both in windows, linux, and Python with `sys.path`.

So maybe `import_paths` makes more sense?

Reply:
I though so too. But those paths are almost always just file paths, not custom loaders. If you want, we can rename it, but I specifically decided against it.

Reply:
Even so, `import_paths` seems more adequate, no?

Reply:
Python uses `sys.path` and `sys.path_hooks` for this. I don't want to split it into multiple arguments.

Ok, I will rename it `import_paths`.